From 7197681335bc8a252e93bf18e5c09f5d845c327f Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Sun, 13 Jun 2021 15:46:35 -0400
Subject: [PATCH 1744/1936] Disable MqTest on mingw

It seems to choke on the <thread> header in some capacity, so disable
for the time being.
---
 src/Tests/CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/Tests/CMakeLists.txt b/src/Tests/CMakeLists.txt
index be6b1e56..cd088696 100644
--- a/src/Tests/CMakeLists.txt
+++ b/src/Tests/CMakeLists.txt
@@ -29,7 +29,6 @@ quick_test(KitTest          ${test_lib})
 quick_test(MemoryStressTest ${test_lib})
 quick_test(MicrotonalTest   ${test_lib})
 quick_test(MsgParseTest     ${test_lib})
-quick_test(MqTest           ${test_lib})
 quick_test(OscilGenTest     ${test_lib})
 quick_test(PadNoteTest      ${test_lib})
 quick_test(RandTest         ${test_lib})
@@ -55,6 +54,10 @@ quick_test(MessageTest zynaddsubfx_core zynaddsubfx_nio
 #Testbed app
 
 if(NOT (${CMAKE_SYSTEM_NAME} STREQUAL "Windows"))
+
+    #std::thread issues with mingw vvvvv
+    quick_test(MqTest           ${test_lib})
+
     add_executable(ins-test InstrumentStats.cpp)
     target_link_libraries(ins-test ${test_lib} rt)
 
-- 
2.47.0

